.github/workflows/on-pullrequest.yml

```diff
@@ -156,7 +156,7 @@ jobs:
             },
             {
               type: "Integration",
-              command: "npm run test:integration -- --ci --config ./tests/config/integration/jest.config.js",
+              command: "npm run test:integration",
```

Comment:
I understand removing the config, but I think we still want `--ci`.

> `--ci`
> When this option is provided, Jest will assume it is running in a CI environment. This changes the behavior when a new snapshot is encountered. Instead of the regular behavior of storing a new snapshot automatically, it will fail the test and require Jest to be run with --updateSnapshot.